std_msgs/Bool request
---
std_msgs/Bool granted
geometry_msgs/Pose pickup_pose
geometry_msgs/Pose dropoff_pose
